app/controllers/agent/manage.go

```diff
@@ -151,13 +153,24 @@ func addVisualExplain(data []byte) ([]byte, error) {
 	}
 	rawExplain := strings.Join(rawExplainRows, "\n")
 	rawExplain = strings.NewReplacer("<nil>", "NULL", "'", "").Replace(rawExplain)
+	rawExplain += "\n"
 
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
-	cmd := exec.CommandContext(ctx, "bash", "-c", "pt-visual-explain <(echo '"+rawExplain+"')")
+	cmd := exec.CommandContext(ctx, "pt-visual-explain")
+	stdin, err := cmd.StdinPipe()
+	if err != nil {
+		return nil, err
+	}
+	go func() {
+		defer stdin.Close()
+		_, err := io.WriteString(stdin, rawExplain)
+		log.Println(err)
```

Comment:
What do you think about, to return this error here?

Reply:
We can't do it easily. Let's leave it.